ledger/ledger.go

```diff
@@ -19,21 +18,17 @@ import (
 
 func LedgerConnection(ledgerAddr, certFile string) (*grpc.ClientConn, error) {
```

Comment:
Why can't you change this to the same as Escrow, Guard, Status client builder?

Whoever's using it (Escrow + go-btfs) will adjust anyway

Reply:
> Why can't you change this to the same as Escrow, Guard, Status client builder?
> 
> Whoever's using it (Escrow + go-btfs) will adjust anyway

working on it, just test that we don't need certFile, will change in next pr.

Reply:
> > Why can't you change this to the same as Escrow, Guard, Status client builder?
> > Whoever's using it (Escrow + go-btfs) will adjust anyway
> 
> working on it, just test that we don't need certFile, will change in next pr.

oh, i see, we can just remove this func, we have LedgerClient in another ledger.go